api/lib/source/polkadotV0-source.js

```diff
@@ -729,20 +732,30 @@ class polkadotAPI {
         ]
       }
     ].concat(
-      proposal.seconds.map((second) => ({
-        depositer: second,
-        amount: [
-          {
-            amount: toViewDenom(this.network, proposal.balance),
-            denom: this.network.stakingDenom
+      Promise.all(
+        proposal.seconds.map(async (second) => {
+          const secondDepositerInfo = await api.derive.accounts.info(second)
+          return {
+            depositer: this.reducers.networkAccountReducer(secondDepositerInfo),
```

Comment:
instead of this being a unique key, why not just `identity` or `networkAccount` for all?